### PR TITLE
Modify rule S110: LaYC format

### PR DESCRIPTION
**Pull request by amelie-renard-sonarsource**

### CODE REVIEW

File: rules/S110/rule.adoc [hunk 1]
```diff
@@ -1,7 +1,5 @@
...
-
-
-This rule raises an issue when the inheritance tree, starting from ``++Object++`` has a greater depth than is allowed.
+Inheritance is one of the most valuable concepts in object-oriented programming. It's a way to categorize and reuse code by creating collections of attributes and behaviors called classes, which can be based on previously created classes.
 
+But abusing this concept by creating a deep inheritance tree can lead to very complex and unmaintainable source code. Most of the time, too deep of an inheritance tree is due to lousy object-oriented design, which leads to systematic use of "inheritance" when "composition" would be better suited.
```
Comment (frederic-tingaud-sonarsource) [inline rules/S110/rule.adoc:5]: For me, "lousy" is a very badly connoted word.
```suggestion
But abusing this concept by creating a deep inheritance tree can lead to complex and unmaintainable source code. Most of the time, an inheritance tree becoming too deep is the symptom of systematic use of "inheritance" when other approaches like "composition" would be better suited.
```

File: rules/S110/php/rule.adoc [hunk 2]
```diff
@@ -1,4 +1,9 @@
 include::../rule.adoc[]
+
+This rule raises an issue when the inheritance tree, starting from `Object`, has a greater depth than is allowed.
```
Comment (frederic-tingaud-sonarsource) [inline rules/S110/php/rule.adoc:3]: Does `Object` exist in PHP?

File: rules/S110/java/rule.adoc [hunk 3]
```diff
@@ -1,25 +1,20 @@
...
 * ``++*++`` matches zero or more characters
 * ``++**++`` matches zero or more packages
 
 Examples:
 
-* ``++java.fwk.AbstractFwkClass++`` will stop count when AbstractFwkClassclass is reached.
-* ``++java.fwk.*++`` will stop count when any member of java.fwkPackage package is reached.
+* ``++java.fwk.AbstractFwkClass++``: the count stops when AbstractFwkClass class is reached.
+* ``++java.fwk.*++``:  any member of java.fwkPackage package is reached.
 * ``++java.fwk.**++`` same as above, but including sub-packages.
```
Comment (frederic-tingaud-sonarsource) [inline rules/S110/java/rule.adoc:15]:
```suggestion
* ``++java.fwk.**++``: same as above, but including sub-packages.
```

**Review by frederic-tingaud-sonarsource — CHANGES_REQUESTED** [review]

File: rules/S110/java/rule.adoc (hunk 3 above)
Comment (amelie-renard-sonarsource) [reply rules/S110/java/rule.adoc:15]: Fixed

File: rules/S110/php/rule.adoc (hunk 2 above)
Comment (amelie-renard-sonarsource) [reply rules/S110/php/rule.adoc:3]: Apparently not, removed

File: rules/S110/rule.adoc (hunk 1 above)
Comment (amelie-renard-sonarsource) [reply rules/S110/rule.adoc:5]: Better, thanks!

**Review by frederic-tingaud-sonarsource — APPROVED** [review]

**Pull request merged**